README.md

```diff
@@ -1,8 +1,54 @@
 # Manual Rover Control System (MRCS)
 A repository for developing the User Interface for receiving data and controlling the rover
+## Packages Info
 
+# Autolaunch
+
+This guide will show you how to set up and launch the MRCS (Manual Rover Control System) using ROS 2 on a Docker bash terminal.
+
+## Required Packages
+1. ROS2 Docker Terminal
+2. Latest version of Tmux: A link to a very useful guide: https://www.hamvocke.com/blog/a-quick-and-easy-guide-to-tmux/
+
+## Instructions
+
+1. Open your ROS 2 Docker terminal.
+2. Create a new directory named `ros2_ws` with a subdirectory called `src` using `mkdir -p ros2_ws/src`.
+3. Navigate to `src` by running `cd ros2_ws/src`.
+4. Clone the latest version of the MRCS repo using `git clone git@github.com:GonzagaRobotics/ManualRoverControlSystem.git`.
+5. Navigate out of `src` in the top of the directory `ros2_ws` by running the command `cd ros2_ws`.
+6. Build the project by running `colcon build`.
+   - If you encounter the error message `stderr: teleop_twist_joy`, run `colcon build` again to resolve it.
+7. Source ROS2 in the parent directory by running the command: `. install/setup.bash`.
+8. Startup `tmux`.
+   - If a green status bar appears at the bottom of your terminal, move on to the next step. Otherwise, try step 9 again, and if that doesn't work, try step 1 to make sure it is installed correctly on your system.
```

Comment:
Change this to say "a green status bar should appear. If it doesn't, make sure you have tmux installed in your docker container and try again"